src/lib/viewers/BaseViewer.js

```diff
@@ -108,6 +109,12 @@ class BaseViewer extends EventEmitter {
         this.options = options;
         this.cache = options.cache;
         this.previewUI = options.ui;
+
+        const fileOptions = options.fileOptions || {};
+        const fileId = getProp(options, 'file.id');
+        const optionsForFileId = fileOptions[fileId] || {};
+        this.fileStartObj = optionsForFileId[FILE_OPTION_START];
```

Comment:
I think you can just do `this.startAt = getProp(options, `fileOptions.${options.file.id}`, {});`

I like keeping `startAt` since that's what we called the Preview-level option.

Reply:
Whatever you decide to name this, add it as a property defined above the constructor. We decided a while back we liked keeping track of all properties and what they're used for at the top of the Class.

Reply:
My intention was to get all the file options and store them, but separating into individual properties on the viewer also makes sense and makes things more readable